dashboard/db/migrate/20190214005506_create_pd_scholarship_infos.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePdScholarshipInfos < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_scholarship_infos do |t|
+      t.belongs_to :user, foreign_key: true, null: false, index: true, dependent: :destroy
```

Comment:
What is the t.belongs_to do?  Is this is similar to "belongs_to"?

Reply:
It creates a `user_id` column. There is a little more info if you search for "belongs_to" in the [migrations documentation](https://guides.rubyonrails.org/active_record_migrations.html) - it's the same thing as "references" if you're familiar with that. In the model, I will have a `belongs_to :user` association to communicate this relationship - so it's similar, but this is a way to express that in the migration, and `belongs_to` is a way to express the relationship in the model.